disco/extensions/upgrade_simulation/upgrades/common_functions.py

```diff
@@ -815,7 +829,10 @@ def get_all_transformer_info_instance(upper_limit=None, compute_loading=True):
     for index, row in all_df.iterrows():
         all_df.at[index, "kVs"] = [float(a) for a in row["kVs"]]
         all_df.at[index, "kVAs"] = [float(a) for a in row["kVAs"]]
-        all_df.at[index, "Xscarray"] = [float(a) for a in row["Xscarray"]]
+        try:

```

Comment:
If you want to check explicitly you can use this. But, you would may need to use something like the semver package to compare versions. Maybe not if you are just checking for 0.6.
```
In [7]: opendssdirect._version.__version__
Out[7]: '0.6.1'
```

Reply:
Well I left this in cause it seemed a weird change from the previous version 
new version: "[1,2,3]"
old version: ["1", "2", "3"]

All the other arrays are still in the old version format -- so I just feel like this is an unusual change, which could be reverted. So just leaving it in there.